tests/ast/data/Dataframe.to_snowpark_pandas.test

```diff
@@ -0,0 +1,186 @@
+## TEST CASE
+
+df = session.table(tables.table1)
+snowpark_pandas_df = df.to_snowpark_pandas()
+
+snowpark_pandas_df = df.to_snowpark_pandas(index_col="A")
+
+snowpark_pandas_df = df.to_snowpark_pandas(columns=["B", "A"])
```

Comment:
Can we add a test case where some pandas op is called after df.to_snowpark_pandas()? We want to make sure nothing breaks then. You can either extend the expectation test here, or add a new integ test under ast for that.

Reply:
done!